webpack.test.config.js

```diff
@@ -22,14 +22,14 @@ module.exports = {
     preLoaders: [
       {
         test: /\.ts$/,
-        loader: 'tslint-loader',
```

Comment:
I left the loader suffix because it's more clear to developers working with webpack for the first time